app/views/questions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="new-question-page">
+  <h1>Create a New Question</h1>
+
+  <div class="errors-partial">
+    <% render "/shared/errors" %>
+  </div>
```

Comment:
If you have to put this div around the render every time you use it, maybe it should be inside the partial.